client/examples/ffi/src/get_auction_info.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+
+#include "casper_client.h"
+
+#define RESPONSE_BUFFER_LEN 1024
+#define ERROR_LEN 255
+
+int main(int argc, char **argv) {
+    const char *node_address = "http://localhost:50101";
+    const char *rpc_id = "1";
+    bool verbose = true;
```

Comment:
would prefer global consts

Reply:
Having `VERBOSE` be a macro would allow setting it at compile time with `-D`